IonicPortals/IonicPortals/PortalsPlugin.swift

```diff
@@ -59,7 +59,7 @@ public class PortalsPlugin: CAPPlugin {
     /// - Returns: A subscription reference to use for unsubscribing
     /// > Tip: Using this method requires you to call ``unsubscribe(_:_:)`` when finished.
     /// Use ``subscribe(to:_:)`` to get an `AnyCancellable` that will automatically unsubscribe from the topic on deallocation.
-    public static func subscribe(_ topic: String, _ callback: @escaping (SubscriptionResult) -> Void) -> Int {
+    @objc public static func subscribe(_ topic: String, _ callback: @escaping (SubscriptionResult) -> Void) -> Int {
```

Comment:
`SubscriptionResult` is a Swift struct, not a class exposed to `@objc`. I've got some work in our backlog to go over the Objective-C API

Reply:
Ok that explains it well thanks. I'll investigate some other ways to message back/forth in meantime.